influxql/ast.go

```diff
@@ -2241,6 +2336,33 @@ func (c *Call) String() string {
 	return fmt.Sprintf("%s(%s)", c.Name, strings.Join(str, ", "))
 }
 
+// Fields will extract any field names from the call.  Only specific calls support this.
+func (c *Call) Fields() []string {
+	switch c.Name {
+	case "top", "bottom":
+		// maintain the sort order
```

Comment:
Again, unless you're actually sorting I would just say "maintain existing order".